models/superrare/superrare_ethereum_events.sql

```diff
@@ -1,6 +1,6 @@
 {{ config(
     alias = 'events',
-    partition_by = ['block_time'],
+    partition_by = ['block_date'],
```

Comment:
another important minor tweak to call out: the partition field on date vs. time

if it's time, then we have a massive amount of directories in data lake with tiny files vs. less directories with larger files. spark prefers the latter. this also matches our base table design on tables such as ```ethereum.transactions```